src/components/CourtDateFilter/CourtDateFilterOptions.tsx

```diff
@@ -20,7 +23,16 @@ const formatNamedDateRange = (namedDateRange: string): string => {
 const labelForDateRange = (namedDateRange: string): string =>
   ["Today", "Yesterday"].includes(namedDateRange) ? namedDateRange : formatNamedDateRange(namedDateRange)
 
-const CourtDateFilterOptions: React.FC<Props> = ({ dateRange, dispatch }: Props) => {
+// TODO- update radio button logic for nested radios. Move away from GOV UK condition classes and use "states" instead.
+const CourtDateFilterOptions: React.FC<Props> = ({ dateRange, dispatch, customDateFrom, customDateTo }: Props) => {
+  const hasCustomDateRange = !!customDateFrom && !!customDateTo
+  const defaultDateValue = (date?: Date | null): string => {
+    if (!!date) {
+      return format(date, "yyyy-MM-dd")
```

Comment:
Would it make sense to create some global variables for the date formats? I've noticed that we have `dateFormatString` in a different file for `dd/MM/yyyy`.

Reply:
The 2nd argument is an input for the `format()` function. I believe the way it is currently laid out is more explicit and readable. 
Perhaps making it into a global variable may make it less readable? @C-gyorfi  @monotypical  do you have any thoughts on this?

Reply:
Agreed to keep as is.